packages/app-project/src/shared/components/StandardLayout/StandardLayout.spec.js

```diff
@@ -62,6 +62,35 @@ describe('Component > StandardLayout', function () {
     expect(adminToggle).to.be.null()
   })
 
+  describe('in beta', function () {
+    before(function () {
+      const snapshot = {
+        project: {
+          beta_approved: true,
+          configuration: {
+            languages: ['en']
+          },
+          launch_approved: false,
+          slug: 'Foo/Bar',
+          strings: {
+            display_name: 'Foobar'
+          },
+          links: {
+            active_workflows: ['1']
+          }
+        }
+      }
+      render(<StandardLayout />, { wrapper: withStore(snapshot)})
+      projectPage = screen.getByTestId('project-page')
+    })
+
+    it('should have a teal border', function () {
+      expect(projectPage).to.be.ok()
+      const { border } = window.getComputedStyle(projectPage)
+      expect(border).to.equal('4px solid #00979d')
+    })
+  })
+
```

Comment:
Should I add a test project with `beta_approved: true` and `launch_approved: true`, to check that launched projects don't have a teal border, even though they are beta-approved? I didn't think of testing the case where the border doesn't show.